### DISCUSSION TITLE
Data consistency concern - Mysterious SKU Mix-ups [Urgent]

### DISCUSSION THREAD

**Discussion by SrdjanPaunovic** [etsy/open-api#1174] [Q&A]
We noticed some weird behavior on listings published via our app using Etsy API v3. It is easier to explain via an example:

We published listing `1607544818` on 2023-11-28 17:02:23.000 +0100 with important SKUs to our system in order.
After that, on the same store, we published `1623501349` on 2023-11-30 18:54:03.000 +0100 with a different set of SKUs.
Later we received an order `3112536645` with the item related to the first listing `1607544818` with the SKU from the listing `1623501349`.

How is this possible to happen? We checked all API calls related to both listings in our logs, and there is no track of updates on listing `1607544818` after 2023-11-28. All the SKUs that we set on Etsy are unique, so is not possible for the listing `1607544818` to contain the SKU that was going to be used for another listing 2 days later. 

This implies that something on Etsy's backend changed the SKUs after we posted the 2nd listing `1623501349` (before that SKUs that are now set on Etsy didn't exist).

Is there a way to check what happened here? Any track of what changed the SKU field? 

This is quite urgent for us because we receive orders with the wrong SKUs, our customers get bad reviews because of that, and we lose money on refund/reprint and our customers.

If you need more information, let me know.

**Comment by Gareth064** [comment]
Hey @SrdjanPaunovic , if you have done all the checks you can on your end to ensure that nothing went wrong there, then the only option is to email Etsy at developer@etsy.com to see if they can check the audit logs at their end.

Note: Etsy don't use this GitHub Discussion board as a way for us to contact them. The only method is via that email address.
